applicationinsights-android/src/main/java/com/microsoft/applicationinsights/internal/TelemetryContext.java

```diff
@@ -107,8 +107,9 @@
      *
      * @param appContext the context for this telemetryContext
      */
-    public TelemetryContext(Context appContext, String instrumentationKey) {
+    public TelemetryContext(Context appContext, String instrumentationKey, String userId) {
 
+        // TODO: Why does everything in here have to be static? Constructor is used to create new instance rather than setting static fields
```

Comment:
This is left over from CLL integration attempt. No reason to keep things static especially with telemetry client as a singleton.